indico/MaKaC/PDFinterface/conference.py

```diff
@@ -376,8 +374,8 @@ def __init__(self, contrib, tz=None):
             'coauthors_affil': coauthor_mapping,
             'contrib': contrib,
             'conf': conf,
-            'tz': conf.getTimezone(),
-            'fields': conf.getAbstractMgr().getAbstractFieldsMgr().getActiveFields()
+            'tz': tz or conf.getTimezone(),
+            'fields': [f for f in conf.as_event.contribution_fields if f.is_active]
```

Comment:
`self._contrib.event_new` - i'd simply assign it to a variable and avoid doing the new-legacy-new "roundtrip"